framework/contracts/native/ibc-client/src/commands.rs

```diff
@@ -251,6 +252,14 @@ pub fn execute_send_module_to_module_packet(
                 .query(&deps.querier, info.sender.clone())?
                 .proxy_address;
             let account_id = cfg.version_control.account_id(&proxy_addr, &deps.querier)?;
+            // Check that ibc_client is installed on account
+            let proxy_state = proxy::state::STATE.query(&deps.querier, proxy_addr)?;
```

Comment:
I would prefer this query to use the manager and check the module there. I don't think the ibc-client is actually registered to allow to execute on the proxy (and if it is we should re-consider that IMO)

Reply:
Removed ibc client and standalone modules from proxy whitelist!